lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -71,14 +71,14 @@ class window.Turbolinks
 
     rememberReferer()
 
-    options.partialReplace ?= false
-    options.onlyKeys ?= []
-    options.onLoadFunction = ->
-      resetScrollPosition() unless options.onlyKeys.length
-      options.callback?()
-
     fetchReplacement url, options
 
+  isPartialReplace = (options, response) ->
+    Boolean(options.partialReplace ||
+    options.onlyKeys?.length ||
+    options.exceptKeys?.length) &&
```

Comment:
This is hard to read with the lack of indentation/ arguments not being lined up.